miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultLibraryTemplateService.java

```diff
@@ -0,0 +1,131 @@
+package uk.ac.bbsrc.tgac.miso.service.impl;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import uk.ac.bbsrc.tgac.miso.core.data.impl.DetailedLibraryTemplate;
+import uk.ac.bbsrc.tgac.miso.core.data.impl.LibraryTemplate;
+import uk.ac.bbsrc.tgac.miso.core.store.DeletionStore;
+import uk.ac.bbsrc.tgac.miso.core.store.LibraryTemplateStore;
+import uk.ac.bbsrc.tgac.miso.core.store.SecurityProfileStore;
+import uk.ac.bbsrc.tgac.miso.core.util.PaginationFilter;
+import uk.ac.bbsrc.tgac.miso.service.LibraryTemplateService;
+import uk.ac.bbsrc.tgac.miso.service.ProjectService;
+import uk.ac.bbsrc.tgac.miso.service.security.AuthorizationManager;
+
+@Transactional(rollbackFor = Exception.class)
+@Service
+public class DefaultLibraryTemplateService implements LibraryTemplateService {
+
+  protected static final Logger log = LoggerFactory.getLogger(DefaultLibraryTemplateService.class);
+
+  @Autowired
+  private LibraryTemplateStore libraryTemplateStore;
+
+  @Autowired
+  private AuthorizationManager authorizationManager;
+
+  @Autowired
+  private SecurityProfileStore securityProfileDao;
+
+  @Autowired
+  private DeletionStore deletionStore;
+
+  @Autowired
+  private ProjectService projectService;
+
+  public void setLibraryTemplateStore(LibraryTemplateStore libraryTemplateStore) {
+    this.libraryTemplateStore = libraryTemplateStore;
+  }
+
+  public void setAuthorizationManager(AuthorizationManager authorizationManager) {
+    this.authorizationManager = authorizationManager;
+  }
+
+  @Override
+  public LibraryTemplate get(long id) throws IOException {
+    authorizationManager.throwIfUnauthenticated();
+    LibraryTemplate libraryTemplate = libraryTemplateStore.get(id);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "libraryTemplate". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)